frontend/src/components/Forms/ShoutoutsPage/ShoutoutCard.tsx

```diff
@@ -2,11 +2,11 @@ import { Card } from 'semantic-ui-react';
 import { Shoutout } from '../../../API/ShoutoutsAPI';
 
 const ShoutoutCard = (props: Shoutout): JSX.Element => {
-  const { giver, receiver, message, isAnon } = props;
+  const { giver, receiver, message, isAnon, timestamp } = props;
 
-  const fromString = isAnon
-    ? 'From: Anonymous'
-    : `From: ${giver?.firstName} ${giver?.lastName} (${giver.email})`;
+  let fromString = isAnon ? 'From: Anonymous' : `From: ${giver?.firstName} ${giver?.lastName}`;
```

Comment:
Feels like the changing this to `let` isn't necessary if we just consolidate these couple lines of template Strings into one. Was the purpose of breaking the code that displays the "from" text up into two lines just for readability?

Reply:
Actually, I think the Date might fit better elsewhere, like the line below the `From: ` text. Combining it with the shoutout author's name feels a little strange.

Reply:
I was also debating putting the Date on the line below 'From:', but I was worried that it would make the page longer and be a lot to scroll through during all-hands. Thoughts on this?

Reply:
Good point. Maybe it could go in the top right corner, horizontally aligned with the `To: ...` text? That seems to be a pretty common place to put it in a lot of online forums.